File: resultat-service/src/main/java/com/backend/resultat_service/model/Examen.java
package com.backend.resultat_service.model;

import lombok.*;

import java.util.Date;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
@ToString
public class Examen {

    private Long id;
    private String nom;
    private Long fkNumDossier;
    private Long fkIdEpreuve;
    private Long fkIdTestAnalyse;
    private Date date;
    private String status;
    private String commentaires;
}

File: src/app/models/examen.ts
export interface Examen {
    id: number;
    fkNumDossier: number;
    fkIdEpreuve: number;
    fkIdTestAnalyse: number;
    date: string;  // Utilisez `string` si vous récupérez la date en format ISO
    status: string;
    commentaires: string;
    nom: string;
}
  
  